cmd/juju/interact/pollster.go

```diff
@@ -399,7 +399,25 @@ func (p *Pollster) queryAdditionalProps(vals map[string]interface{}, schema *jso
 		// We assume that the name of the schema is the name of the object the
 		// schema describes, and for additional properties the property name
 		// (i.e. map key) is the "name" of the thing.
-		name, err := p.EnterVerify(schema.Singular+" name", verifyName)
+		var name string
+		var err error
+		if schema.Default != nil {
+			var def string
+			if schema.PromptDefault != nil {
+				def = fmt.Sprintf("%v", schema.PromptDefault)
```

Comment:
any chance you want to add allowing for an env var here as with pr 9015?  then we can uncomment provider/openstack/provider.go: 126 and remove the TODO.  :-D 

                        if schema.PromptDefault != nil {
                                def = fmt.Sprintf("%v", schema.PromptDefault)
                        }
                        var defFromEnvVar string
                        if len(schema.EnvVars) > 0 {
                                for _, envVar := range schema.EnvVars {
                                        value := os.Getenv(envVar)
                                        if value != "" {
                                                defFromEnvVar = value
                                                def = defFromEnvVar
                                                break
                                        }
                                }
                        }
                        if def == "" {
                                def = fmt.Sprintf("%v", schema.Default)
                        }

and a change at line 415 perhaps too.

Reply:
Let's do that in another PR